### PR TITLE
Add AUTOSCALING_LOGLEVEL

### PR DESCRIPTION
**Pull request by mrnicegyu11**
Add env-var AUTOSCALING_LOGLEVEL

Goes with https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/142

### CODE REVIEW

**Comment by mrnicegyu11** [comment]
> Is it the right place? Not in `osparc-simcore`?

@YuryHrytsuk that could be, you are the expert so you tell me: 

In osparc-simcore we already have:
```
  autoscaling:
    image: ${DOCKER_REGISTRY:-itisfoundation}/autoscaling:${DOCKER_IMAGE_TAG:-latest}
    init: true
    hostname: "{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"
    networks:
      - autoscaling_subnet
    environment:
      - LOG_LEVEL=${LOG_LEVEL:-WARNING}
```

so it captures the "global" `LOG_LEVEL`, but i want to be able to set the AUTOSCALER's loglevel specifically to DEBUG for ... debugging. What is the proper way to do this? :)

**Comment by sanderegg** [comment]
> > Is it the right place? Not in `osparc-simcore`?
> 
> @YuryHrytsuk that could be, you are the expert so you tell me:
> 
> In osparc-simcore we already have:
> 
> ```
>   autoscaling:
>     image: ${DOCKER_REGISTRY:-itisfoundation}/autoscaling:${DOCKER_IMAGE_TAG:-latest}
>     init: true
>     hostname: "{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"
>     networks:
>       - autoscaling_subnet
>     environment:
>       - LOG_LEVEL=${LOG_LEVEL:-WARNING}
> ```
> 
> so it captures the "global" `LOG_LEVEL`, but i want to be able to set the AUTOSCALER's loglevel specifically to DEBUG for ... debugging. What is the proper way to do this? :)

Actually @mrnicegyu11 @YuryHrytsuk we should find a way to set a LOG_LEVEL to separate services. because, even using AUTOSCALING_LOG_LEVEL, how will set a different log level for different instances of Autoscaling. For example in the case where we would have one for s4l-lite and another for sim4life ?

File: services/simcore/docker-compose.yml [hunk 1]
```diff
@@ -45,6 +45,7 @@ services:
       - NODES_MONITORING_NODE_LABELS=${AUTOSCALING_NODES_MONITORING_NODE_LABELS} # defines the labels to check for on the nodes (i.e. ["dynamicsidecar"])
       - NODES_MONITORING_SERVICE_LABELS=${AUTOSCALING_NODES_MONITORING_SERVICE_LABELS} # optional defines the label to check for on the monitored services
       - NODES_MONITORING_NEW_NODES_LABELS=${NODES_MONITORING_NEW_NODES_LABELS} # optional defines the name of the monitored service
+      - LOG_LEVEL=${AUTOSCALING_LOGLEVEL}
```
Comment (matusdrobuliak66) [inline services/simcore/docker-compose.yml:48]:
```suggestion
      - AUTOSCALING_LOGLEVEL=${AUTOSCALING_LOGLEVEL}
```

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by YuryHrytsuk — APPROVED** [review]

**Review by sanderegg — APPROVED** [review]
my comment still hold I guess ;)

**Comment by mrnicegyu11** [comment]
> my comment still hold I guess ;)

Yes, but I have no clue how yet :/

**Comment by sanderegg** [comment]
@mrnicegyu11 I think you can disable that PR. I will replace it completely and autoscaling_log_level will come in directly in osparc-simcore.
Also I will issue a PR where the autoscaling here gets simpler (as are the other services.)
for info see -> https://github.com/ITISFoundation/osparc-simcore/pull/5033

**Pull request closed without merging**